Framework/src/runBasic.cxx

```diff
@@ -124,3 +127,17 @@ WorkflowSpec defineDataProcessing(const ConfigContext& config)
 
   return specs;
 }
+
+// TODO merge this with the one from runReadout.cxx
```

Comment:
I wouldn't sweat it, because when we use the workflow merging, in runBasic and runReadout we will have data producers only (nothing to configure) and the config files will need to be passed explicitly to o2-qc-run-qc

Reply:
Ok, so I will not press this issue.